test/Mono.Linker.Tests.Cases/Reflection/ExpressionCallString.cs

```diff
@@ -253,11 +254,10 @@ static void TestMethodWithoutRequirements ()
 			}
 
 			[Kept]
-			[ExpectedWarning ("IL2060", "Expression::Call")]
 			static void TestMethodWithRequirements ()
 			{
-				// This must warn - as this is dangerous
-				Expression.Call (typeof (TestGenericMethods), nameof (GenericMethodWithRequirements), new Type[] { GetUnknownType () });
+				// This may not warn - as it's safe
+				Expression.Call (typeof (TestGenericMethods), nameof (GenericMethodWithRequirements), new Type[] { GetUnknownTypeWithRequrements () });
```

Comment:
Can you keep both cases? Otherwise I don't see a case where we get an unknown type in the array.

Reply:
I'll add in a new case that generates the correct "cannot be analyzed" semantics.